src/pages/auth/Login.jsx

```diff
@@ -0,0 +1,44 @@
+import "./auth.css";
+import {Link} from "react-router-dom";
+import {useState} from "react";
+
+export function Login(){
+    const [visibility, setVisibility] = useState(false);
+    
+    const showHideHandler = (e) => {
```

Comment:
suggestion, can think of better name, eg: visibilityHandler

Reply:
Thank you for your suggestion, I have updated it.